src/Sylius/Component/Order/Model/Order.php

```diff
@@ -435,7 +435,9 @@ public function getAdjustmentsTotal($type = null)
 
         $total = 0;
         foreach ($this->getAdjustments($type) as $adjustment) {
-            $total += $adjustment->getAmount();
+            if (!$adjustment->isNeutral()) {
+                $total += $adjustment->getAmount();
+            }
```

Comment:
But we also need to obtain the adjustments total taking into account the neutrality. E.g.:
```
$shippingTotal = $this->getAdjustmentsTotal(AdjustmentInterface::SHIPPING_ADJUSTMENT);
$shippingTotal += $this->getAdjustmentsTotal(AdjustmentInterface::ORDER_PROMOTION_ADJUSTMENT);
$shippingTotal += $this->getAdjustmentsTotal(AdjustmentInterface::TAX_ADJUSTMENT); // if tax is already included in shipping fee I don't want to add it once again
```

If we will make `getAdjustmentsTotal` method that will return only sum of non neutral adjustments and then we will have `getAdjustmentsTotalRecursively` that will return sum of all adjustments (both neutral and non neutral) it will be confusing.

Right now `getAdjustmentsTotal()` will return sum of only non neutral adjustments on current level, but `getAdjustmentsTotal($type)` will return sum of all adjustments on current level. This can be confusing. 

How about renaming method `getAdjustmentsTotal` to `getNonNeturalAdjustmentsTotal` and leave current logic of `getAdjustmentsTotalRecursively` and add `getNonNeturalAdjustmentsTotalRecursively` method if needed?